projects/optic/src/__tests__/integration/__snapshots__/history.test.ts.snap

```diff
@@ -4,93 +4,93 @@ exports[`optic history writes changelog 1`] = `
-  - removed \`requestBody\` \`items\`.\`phone\` property
-  - added \`requestBody\` \`items\`.\`bio\` property
-  - changed \`requestBody\` \`items\`.\`userStatus\`.\`type\`
-  - removed \`requestBody\` \`items\`.\`userStatus\`.\`format\`
-  - added \`requestBody\` \`items\`.\`userStatus\`.\`enum\`
-  - removed \`requestBody\` \`items\`.\`lastName\` as required
-  - added \`requestBody\` \`items\`.\`username\` as required
+  - \`requestBody\`: removed support for \`items\`.\`phone\` property
+  - \`requestBody\`: added support for \`items\`.\`bio\` property
+  - \`requestBody\`: changed \`items\`.\`userStatus\`.\`type\`
+  - \`requestBody\`: removed \`items\`.\`userStatus\`.\`format\`
+  - \`requestBody\`: added \`items\`.\`userStatus\`.\`enum\`
+  - \`requestBody\`: \`items\`.\`lastName\` is not required anymore
+  - \`requestBody\`: \`items\`.\`username\` is now required
 - \`POST\` \`/user\`:
-  - removed \`requestBody\` \`phone\` property
-  - added \`requestBody\` \`bio\` property
-  - changed \`requestBody\` \`userStatus\`.\`type\`
-  - removed \`requestBody\` \`userStatus\`.\`format\`
-  - added \`requestBody\` \`userStatus\`.\`enum\`
-  - removed \`requestBody\` \`lastName\` as required
-  - added \`requestBody\` \`username\` as required
-- \`DELETE\` \`/store/order/{orderId}\`: added \`404\` response
+  - \`requestBody\`: removed support for \`phone\` property
+  - \`requestBody\`: added support for \`bio\` property
+  - \`requestBody\`: changed \`userStatus\`.\`type\`
+  - \`requestBody\`: removed \`userStatus\`.\`format\`
+  - \`requestBody\`: added \`userStatus\`.\`enum\`
+  - \`requestBody\`: \`lastName\` is not required anymore
+  - \`requestBody\`: \`username\` is now required
+- \`DELETE\` \`/store/order/{orderId}\`: \`404\` response: added \`application/json\`
 - \`GET\` \`/store/order/{orderId}\`:
-  - added \`404\` response
-  - added \`200\` response \`summary\` property
-  - removed \`200\` response \`status\`.\`enum\`.\`1\`
-  - added \`200\` response \`status\`.\`enum\`.\`2\`
-  - removed \`200\` response \`petId\` as required
+  - \`404\` response: added \`application/json\`
+  - \`200\` response: added support for \`summary\` property
+  - \`200\` response: removed support for value \`approved\` on enum \`status\`
+  - \`200\` response: added support for new value \`canceled\` on enum \`status\`
+  - \`200\` response: \`petId\` is not required anymore
 - \`POST\` \`/store/order\`:
-  - added \`200\` response \`summary\` property
-  - removed \`200\` response \`status\`.\`enum\`.\`1\`
-  - added \`200\` response \`status\`.\`enum\`.\`2\`
-  - added \`requestBody\` \`summary\` property
-  - removed \`requestBody\` \`status\`.\`enum\`.\`1\`
-  - added \`requestBody\` \`status\`.\`enum\`.\`2\`
+  - \`200\` response: added support for \`summary\` property
+  - \`200\` response: removed support for value \`approved\` on enum \`status\`
+  - \`200\` response: added support for new value \`canceled\` on enum \`status\`
```

Comment:
hmmm I feel like it's `added enum option 'canceled' on 'status'` and `removed enum option approved on 'status'`

Reply:
I can change but just to let you know I took the Stripe wording there